nginx/nginx.go

```diff
@@ -208,8 +206,6 @@ func (n *nginxUpdater) ensureNginxRunning() error {
 
 		go n.periodicallyUpdateMetrics()
 		go n.backgroundSignaller()
-
-		n.initialUpdateApplied.done = true
```

Comment:
With it in its previous position, it wasn't possible for the healthcheck to tell the difference between waiting for an initial update to be applied, and Nginx having stopped unexpectedly.

That's maybe a lesser concern, however, and we can find some other way of conveying that information.